config/checkstyle_checks.xml

```diff
@@ -444,6 +444,7 @@
     <module name="UnnecessarySemicolonInEnumeration"/>
     <module name="UnnecessarySemicolonInTryWithResources"/>
     <module name="VariableDeclarationUsageDistance"/>
+    <module name="GenericXpath"/>
```

Comment:
We need to find better use for us.

Reply:
not done, 
we need Check to work on our code base.
default now do nothing.

Reply:
maybe remove it at all? 

Or we can create custom checks, do we have list of checks we want to implement, but they haven't been implemented yet? so that we can mimic them

Reply:
done, added check for `printStackTrace`